### PR TITLE
6.4.0 helm intallation fix

### PR DESCRIPTION
**Pull request by ChomickiM**
Helm deployments were failing due to the liquibase changes requiring the "uuid-ossp" extension. To resolve this, we've moved this step into a script that is run inside of the docker-entrypoint before the liquibaseChangelockReset. This created a new problem where the commands can only be run by an admin user. We are now looking for the postgres admin usernames and password and need to add additional values into the yaml file in order to get these values.

### CODE REVIEW

**Review by gkillough — APPROVED** [review]

**Review by bamandel — APPROVED** [review]

**Review by jamesrichard91 — APPROVED** [review]

**Review by psantos1113 — APPROVED** [review]

File: deployment/helm/synopsys-alert/README.md [hunk 1]
```diff
@@ -452,6 +455,46 @@ For the on-premise database deployment a second Persistent Volume must be create
     databaseName: <DATABASE_NAME>
   ```
   - Replace <DATABASE_NAME> with the name of the database created in previous steps
+
+- Configure Postgres admin username and password following the instructions outlined here: [Using External Database](#using-external-database)
```
Comment (jamesrichard91) [inline deployment/helm/synopsys-alert/README.md:459]: Should there be a similar message on line 394 linking to the internal database instructions?

Comment (psantos1113) [reply deployment/helm/synopsys-alert/README.md:459]: I agree we should like to the internal database instructions.

File: deployment/helm/synopsys-alert/README.md [hunk 2]
```diff
@@ -452,6 +455,46 @@ For the on-premise database deployment a second Persistent Volume must be create
...
+
+### Database Admin User Password
+We now require administrator access to configure the database correctly. The following section describes how to configure the administrator credentials for both internal and external database configurations.
+
+#### Using Internal Database
```
Comment (psantos1113) [inline deployment/helm/synopsys-alert/README.md:464]: Change internal to On Premise to be consistent.  Update any links to this section

File: deployment/helm/synopsys-alert/README.md [hunk 3]
```diff
@@ -452,6 +455,46 @@ For the on-premise database deployment a second Persistent Volume must be create
     databaseName: <DATABASE_NAME>
   ```
   - Replace <DATABASE_NAME> with the name of the database created in previous steps
+
+- Configure Postgres admin username and password following the instructions outlined here: [Using External Database](#using-external-database)
+
+### Database Admin User Password
+We now require administrator access to configure the database correctly. The following section describes how to configure the administrator credentials for both internal and external database configurations.
```
Comment (psantos1113) [inline deployment/helm/synopsys-alert/README.md:462]: Change the word internal to on premise to be consistent.

File: deployment/helm/synopsys-alert/README.md [hunk 4]
```diff
@@ -452,6 +455,46 @@ For the on-premise database deployment a second Persistent Volume must be create
...
+#### Using Internal Database
+  - While editing the alert values.yaml, in the postgres section edit the following configurations to set the admin user password of the postgres database.
+  
+    ```yaml
+    postgres:
+      adminUserName: postgres
+      adminPassword: "<ADMIN_PASSWORD>"
+    ```
+    - Replace <ADMIN_PASSWORD> with the name of the postgres administrator password
+        - Note: The adminUserName: value must remain "postgres" when using the internal postgres database.
```
Comment (psantos1113) [inline deployment/helm/synopsys-alert/README.md:473]: change internal to on premise

**Review by psantos1113 — APPROVED** [review]

**Pull request merged**